packages/story-editor/src/utils/useDoubleClick.ts

```diff
@@ -19,6 +19,8 @@
  */
 import { useCallback, useEffect, useState } from '@googleforcreators/react';
 
+type clickCallback = (evt: Event, target: Element) => void;
```

Comment:
Nit: let's capitalize our types for consistency. I'll see if we can enforce this with the linter

Also, can we perhaps narrow down the type a bit more?

This should be a `MouseEvent`.

Perhaps can use `MouseEventHandler<HTMLElement>` too

Reply:
Fixed.